entity-framework/core/miscellaneous/cli/dotnet.md

```diff
@@ -121,6 +121,12 @@ Entity Framework Core .NET Command-line Tools 2.1.3-rtm-32065
 <Usage documentation follows, not shown.>
 ```
 
+
+## Updating the tools
+
+Use `dotnet tool update --global dotnet-ef` to update the tools to the latest available version, install a specific version by appending `--version <VERSION>` to your command.
```

Comment:
It should differentiate between updating global tool vs local tool.

Reply:
What is a local tool and how do you update it?

Reply:
Is it referring to running `dotnet tool update dotnet-ef` without the --global parameter?

Reply:
https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools